src/EnergyPlus/api/datatransfer.py

```diff
@@ -265,8 +265,7 @@ def api_data_fully_ready(self, state: c_void_p) -> bool:
         :param state: An active EnergyPlus "state" that is returned from a call to `api.state_manager.new_state()`.
         :return: Returns a boolean value to indicate whether variables, actuators, and other data are ready for access.
         """
-        success = self.api.apiDataFullyReady(state)
-        if success == 0:
+        if self.api.apiDataFullyReady(state) == 1:
```

Comment:
I do like that the change is concealed inside the Python code so that the Python API doesn't change.  I would say that of the API consumers, it's surely like > 90% or 95%, maybe more.  We get very few questions about the C API, and I give almost zero demos/discussions about the C layer.  This definitely makes me feel better about this change.